tailwind.config.js

```diff
@@ -2,7 +2,11 @@
 module.exports = {
   content: ['./index.html', './src/**/*.{vue,js,ts,jsx,tsx}'],
   theme: {
-    extend: {},
+    extend: {
```

Comment:
Not a big deal, but it's better to generally split fixes into different commits if they come in same PR. On review we then check commit by commit instead of all changes at once, which helps with focus.